pages/SpotInformation/[id].js

```diff
@@ -2,13 +2,15 @@ import Header from "../../components/Header";
 import SpotInfo from "../../components/SpotInfo";
 import { useRouter } from "next/router";
 import Navigation from "../../components/Nav";
+import LoadingSpinner from "../../components/LoadingSpinner";
+
 
 export default function SpotInformation() {
   const router = useRouter();
   const { id } = router.query;
 
   if (!id) {
-    return <h2>Spot not found</h2>;
+    return <LoadingSpinner />;;
```

Comment:
Why do you render a loading spinner when the spot does not exist? So the spinner will spin and spin and spin...